bin/www

```diff
@@ -4,49 +4,49 @@
  * Module dependencies.
```

Comment:
What is the reason for all the changes to this file?

Reply:
I just ran the >ESLint: Fix all auto-fixable Problems command

Reply:
No usar el autofix o el lint fix porque fixea todo lo que encuentra y modifica sin necesidad archivos que no estan en el scope de la tarea